src/styles/blocks/calendar.scss

```
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin: auto;
+  width: 710px;
+  gap: $gap-between-days;
+
+  $week-days:
+    "mon" 0,
+    "tues" $box-size + $gap-between-days,
+    "wed" 2 * ($box-size + $gap-between-days),
+    "thurs" 3 * ($box-size + $gap-between-days),
+    "fri" 4 * ($box-size + $gap-between-days),
+    "sat" 5 * ($box-size + $gap-between-days),
+    "sun" 6 * ($box-size + $gap-between-days);
+
+  @each $day, $margin in $week-days {
+    &--day-#{$day} &__day--1 {
```

use pseudoclass here
```suggestion
    &--day-#{$day}  {
```